subworkflows/local/prepare_references.nf

```diff
@@ -153,6 +161,7 @@ workflow PREPARE_REFERENCES {
         gnomad_af_idx         = TABIX_GNOMAD_AF.out.tbi.collect()                                // channel: [ val(meta), path(fasta) ]
         known_dbsnp_tbi       = TABIX_DBSNP.out.tbi.collect()                                    // channel: [ val(meta), path(fasta) ]
         target_bed            = Channel.empty().mix(ch_tbi, ch_bgzip_tbi).collect()              // channel: [ val(meta), path(bed), path(tbi) ]
+        vcfanno_extra         = ch_vcfanno_extra.ifEmpty([[]])   // channel: [ path(intervals) ]
```

Comment:
do we need a collect here to make it a value channel?

Reply:
I have added a collect statement where I create the channel, and ifEmpty returns a value channel if the source channel is also a value so it should work.